specification/eventgrid/data-plane/Microsoft.PolicyInsights/stable/2018-01-01/PolicyInsights.json

```diff
@@ -0,0 +1,113 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-01-01",
+    "title": "Schema of Azure Policy events published to Azure Event Grid",
+    "description": "Describes the schema of the Azure Policy events published to Azure Event Grid. This corresponds to the Data property of an EventGridEvent."
+  },
+  "paths": {},
+  "definitions": {
+    "PolicyInsightsPolicyStateCreatedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.PolicyInsights.PolicyStateCreated event.",
+      "type": "object",
+      "properties": {
+        "timestamp": {
+          "description": "The time (in UTC) that the resource was scanned by Azure Policy.",
+          "type": "string"
```

Comment:
Is this a ISO8601 or RFC format that could be precised a bit more?

Reply:
Updated to specify ISO8601.